zebrad/src/components/mempool.rs

```diff
@@ -141,6 +148,11 @@ impl Service<Request> for Mempool {
                 let _ = self.storage.insert(tx);
             }
         }
+
+        if let Some(tip_height) = self.latest_chain_tip.best_tip_height() {
+            let _ = remove_expired_transactions(&mut self.storage, tip_height);
```

Comment:
Nitpick: there's no return value here any more.

```suggestion
            remove_expired_transactions(&mut self.storage, tip_height);
```